src/plugins/dbg/quictypes.h

```diff
@@ -1429,6 +1458,21 @@ struct Registration : Struct {
     String GetAppName() {
         return String(AddrOf("AppName"));
     }
+
+    PSTR GetWorkersState() {
+        auto Workers = GetWorkerPool();
+        UCHAR WorkerCount = Workers.WorkerCount();
+        bool HasQueuedWorker = false;
+        for (UCHAR i = 0; i < WorkerCount; i++) {
+            if (Workers.GetWorker(i).IsActive()) {
+                return " ACTIVE";
+            }
+            if (Workers.GetWorker(i).HasWorkQueue()) {
+                HasQueuedWorker = true;
```

Comment:
why not `return "ACTIVE"; `?

Reply:
Because the thread isn't actually running yet. It's just queued. But if any thread is running, I want to mark the whole registration as active.